api/database/migrations/2022_03_03_231414_update_users_table_with_profile_data.php

```diff
@@ -0,0 +1,109 @@
+            $table->string('comprehension_level')->nullable();
+            $table->string('written_level')->nullable();
+            $table->string('verbal_level')->nullable();
+            $table->string('estimated_language_ability')->nullable();
+            $table->boolean('is_gov_employee')->nullable();
+            $table->boolean('interested_in_later_or_secondment')->nullable();
+            $table->uuid('current_classification')->nullable();
+            $table->foreign('current_classification')->references('id')->on('classifications');
+            $table->jsonb('location_exemptions')->nullable();
+            $table->boolean('would_accept_temporary')->nullable();
+
+            // Old fields from poolCandidate
+            $table->boolean('is_woman')->nullable();
+            $table->boolean('has_disability')->nullable();
+            $table->boolean('is_indigenous')->nullable();
+            $table->boolean('is_visible_minority')->nullable();
+
+            $table->boolean('has_diploma')->nullable();
+            $table->string('language_ability')->nullable();
+            $table->jsonb('location_preferences')->nullable();
+            $table->jsonb('expected_salary')->nullable();
+        });
+
+        // New tables to replace connections with pool candidates
+        Schema::create('operational_requirement_user', function (Blueprint $table) {
+            $table->uuid('id')->primary('id');
+            $table->timestamps();
+            $table->uuid('user_id')->references('id')->on('users')->nullable(false);
+            $table->uuid('operational_requirement_id')->references('id')->on('operational_requirements')->nullable(false);
+        });
+        DB::statement('ALTER TABLE operational_requirement_user ALTER COLUMN id SET DEFAULT gen_random_uuid();');
+        Schema::create('classification_user', function (Blueprint $table) {
+            $table->uuid('id')->primary('id');
+            $table->timestamps();
+            $table->uuid('user_id')->references('id')->on('users')->nullable(false);
+            $table->uuid('classification_id')->references('id')->on('classifications')->nullable(false);
+        });
+        DB::statement('ALTER TABLE classification_user ALTER COLUMN id SET DEFAULT gen_random_uuid();');
+        Schema::create('cmo_asset_user', function (Blueprint $table) {
+            $table->uuid('id')->primary('id');
+            $table->timestamps();
+            $table->uuid('user_id')->references('id')->on('users')->nullable(false);
+            $table->uuid('cmo_asset_id')->references('id')->on('cmo_assets')->nullable(false);
+        });
+        DB::statement('ALTER TABLE cmo_asset_user ALTER COLUMN id SET DEFAULT gen_random_uuid();');
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->dropColumn([
```

Comment:
Try rolling the migration back (`php artisan migrate:rollback --step=1`) and forward.  The column `location_exemptions` is missing from the down function.